.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "6"
```

Comment:
Well the purpose of using multiple versions is ensuring we support the app for multiple versions of node. NOT from the point of view how fast the test runs. Also v6 is the current LTS version so I thought of using it.